src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -17,15 +17,30 @@ namespace Microsoft.OpenApi.OData
     /// </summary>
     public class OpenApiConvertSettings
     {
+        private Version _version = new(1, 0, 1);
         /// <summary>
         /// Gets/sets the service root.
         /// </summary>
         public Uri ServiceRoot { get; set; } = new Uri("http://localhost");
 
         /// <summary>
-        /// Gets/sets the metadata version.
+        /// Get/set the metadata version.
         /// </summary>
-        public Version Version { get; set; } = new Version(1, 0, 1);
+        [Obsolete("Use SemVerVersion to Get or Set the metadata version.")]
+        public Version Version 
+        { 
+            get => _version; 
+            set 
+            {
+                _version = value;
+                SemVerVersion = _version.ToString();
+            }
+        }
```

Comment:
```suggestion
        } = new(1, 0, 1);
```

Reply:
Only auto-implemented properties can have initializers.
Decided to implement it like this instead:

```CSharp
public Version Version 
{ 
    get => Version.TryParse(SemVerVersion, out var version) ? version : null; 
    set 
    {
        SemVerVersion = value?.ToString() ?? "1.0.1";
    }
}
```